task2/src/main/java/neutron/Neutron.java

```diff
@@ -0,0 +1,47 @@
+package neutron;
```

Comment:
https://pl.wikipedia.org/wiki/Neuron
https://pl.wikipedia.org/wiki/Neutron
It's not the same. Name should Neuron.